components/Item/Card/Card.vue

```diff
@@ -120,10 +120,10 @@ export default Vue.extend({
         return false;
       }
     },
-    overlay: {
+    noOverlay: {
```

Comment:
I would prefer we leave it as `overlay` and set the default to false

Reply:
We're switching most of the props (see https://github.com/jellyfin/jellyfin-vue/pull/592#issuecomment-762050298)

Reply:
I don't quite understand, The comment, on 592 is what I'm saying?

Reply:
MrTimscampi says that, if a property is not specified, it's false by default. Before, we were defaulting overlay to ``true`` and, for disabling it, we needed to pass it like ``:overlay="false"``, which is counter-intuitive. 

Other props in card also follow the approach this prop will follow now: they are like ``noX`` and setting them means they're false, no need to do the ``:overlay="false"`` thing anymore, just ``no-overlay`` when using the component.

Reply:
I think Cam is just saying to keep the property as `overlay` even though it is at false by default (which I agree should be)

Reply:
Ah I didn't notice that all the others were `noX`, imho, like vuetify does, we should prefer the other method (instead of `noOverlay` have `overlay`). Since we do this elsewhere we can leave for now. Thoughts @ThibaultNocchi

Reply:
I think Cam is just saying to keep the property as `overlay` even though it is at false by default (which I agree should be)

Reply:
So we will leave as is for now, and refactor later. @ThibaultNocchi one last approval before merge?